views/locations/locations_archive.php

```diff
@@ -1,64 +1,121 @@
 <?php
-/**
- * PEP Capping 2017 Algozzine's Class
```

Comment:
Any reason to remove header comment here?

Reply:
I think its cause I renamed the files around (switched `_archive` and `_delete`) but never put a header on the other file so it looks like I removed it. These pages aren't actually used either since "locations" don't exist anymore.